package.json

```diff
@@ -40,16 +40,16 @@
   },
   "devDependencies": {
     "@types/jest": "27.0.3",
-    "@types/node": "16.11.12",
+    "@types/node": "17.0.10",
```

Comment:
for better stability I would recommend updating the built-test-lint.yaml workflow to use node 17, as well as updating docs and other workflows that mention the node version.

also this package.json file should mention the node version under the engines block to match the node types that you want to use.

Reply:
Done. But @alzafacon will you check that I got the "engines" version right.

Reply:
package.json looks good, but the node-version in built-test-lint.yaml needs to be more specific
node-version: '17.0.10'

I think that's why the workflow is failing on yarn install